pkgs/development/python-modules/openai/default.nix

```diff
@@ -81,10 +90,7 @@ buildPythonPackage rec {
 
   disabledTests = [
     # Tests make network requests
-    "test_streaming_response"
     "test_copy_build_request"
-
-    # Test fails with pytest>=8
```

Comment:
Why removing this comment?

Reply:
It wasn't true (anymore, at least as far as I can tell) but it does seem to require network access, since the error message says it needs an API key.